templates/todo/projects/nodejs-mongo/infra/resources.bicep

```diff
@@ -189,89 +182,17 @@ module applicationInsightsResources '../../../../common/infra/applicationinsight
   }
 }
 
-resource cosmos 'Microsoft.DocumentDB/databaseAccounts@2021-10-15' = {
-  name: '${abbrs.documentDBDatabaseAccounts}${resourceToken}'
-  kind: 'MongoDB'
-  location: location
-  tags: tags
-  properties: {
-    consistencyPolicy: {
-      defaultConsistencyLevel: 'Session'
-    }
-    locations: [
-      {
-        locationName: location
-        failoverPriority: 0
-        isZoneRedundant: false
-      }
-    ]
-    databaseAccountOfferType: 'Standard'
-    enableAutomaticFailover: false
-    enableMultipleWriteLocations: false
-    apiProperties: {
-      serverVersion: '4.0'
-    }
-    capabilities: [
-      {
-        name: 'EnableServerless'
-      }
-    ]
-  }
-
-  resource database 'mongodbDatabases' = {
-    name: 'Todo'
-    properties: {
-      resource: {
-        id: 'Todo'
-      }
-    }
-
-    resource list 'collections' = {
-      name: 'TodoList'
-      properties: {
-        resource: {
-          id: 'TodoList'
-          shardKey: {
-            _id: 'Hash'
-          }
-          indexes: [
-            {
-              key: {
-                keys: [
-                  '_id'
-                ]
-              }
-            }
-          ]
-        }
-      }
-    }
-
-    resource item 'collections' = {
-      name: 'TodoItem'
-      properties: {
-        resource: {
-          id: 'TodoItem'
-          shardKey: {
-            _id: 'Hash'
-          }
-          indexes: [
-            {
-              key: {
-                keys: [
-                  '_id'
-                ]
-              }
-            }
-          ]
-        }
-      }
-    }
+module cosmosResources '../../../common/infra/modules/cosmos.bicep' = {
```

Comment:
Once `repoman` has done it's work, this gets rewritten to: 

```
module cosmosResources 'infra/modules/cosmos.bicep' = {
```

https://github.com/Azure-Samples/todo-nodejs-mongo/compare/main...pr/485#diff-e800ed75064b740be458a572b91420b94ca6743a609c7318facf638f60c7028bR185

But I think it needs to be just:

```
module cosmosResources './modules/cosmos.bicep' = {
```

since `resources.bicep` ends up in the `infra` folder.

I pulled the PR for the generated template and tried to build the resulting bicep and got errors:

```
C:\dd\azure-samples\todo-nodejs-mongo\infra\resources.bicep(80,35) : Error BCP062: The referenced declaration with name "cosmosResources" is not valid.
C:\dd\azure-samples\todo-nodejs-mongo\infra\resources.bicep(185,24) : Error BCP091: An error occurred reading file. Could not find a part of the path 'C:\dd\azure-samples\todo-nodejs-mongo\infra\infra\modules\cosmos.bicep'.
C:\dd\azure-samples\todo-nodejs-mongo\infra\resources.bicep(194,52) : Error BCP062: The referenced declaration with name "cosmosResources" is not valid.
C:\dd\azure-samples\todo-nodejs-mongo\infra\resources.bicep(195,44) : Error BCP062: The referenced declaration with name "cosmosResources" is not valid.
C:\dd\azure-samples\todo-nodejs-mongo\infra\main.bicep(25,18) : Error BCP104: The referenced module has errors.
```

Reply:
Yes, thanks for testing.  I'm pushed this so Wallace could have a look :)